osu.Game/Overlays/WikiOverlay.cs

```diff
@@ -25,15 +25,21 @@ public class WikiOverlay : OnlineOverlay<WikiHeader>
         [Resolved]
         private IAPIProvider api { get; set; }
 
+        [Cached]
+        private readonly OverlayScrollContainer scrollContainer;
+
         private GetWikiRequest request;
 
         private CancellationTokenSource cancellationToken;
 
         private bool displayUpdateRequired = true;
 
+        private WikiArticlePage articlePage;
+
         public WikiOverlay()
             : base(OverlayColourScheme.Orange, false)
         {
+            scrollContainer = ScrollFlow;
```

Comment:
This seems a little dangerous as it relies on the base initialising everything in ctor and not in BDL, for example. Think I'll just remove this field and hoist the `[Cached]` spec up to the original place of definition. Seems like a generally useful thing to cache anyways.